ast/functions_test.go

```diff
@@ -1,84 +0,0 @@
-// Copyright 2016 PingCAP, Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package ast
-
-import (
-	. "github.com/pingcap/check"
-	"github.com/pingcap/tidb/model"
-)
-
-var _ = Suite(&testFunctionsSuite{})
-
-type testFunctionsSuite struct {
-}
-
-func (ts *testFunctionsSuite) TestAggregateFuncExtractor(c *C) {
```

Comment:
No longer, we remove the FirstRow logic and extractor is quite simple now. I think we can abandon it in future.